packages/pg/test/integration/gh-issues/1105-tests.js

```diff
@@ -0,0 +1,20 @@
+const pg = require('../../../lib')
+const helper = require('../test-helper')
+const suite = new helper.Suite()
+
+suite.testAsync('timeout causing query crashes', async () => {
+  const client = new helper.Client()
+  await client.connect()
+  await client.query('CREATE TEMP TABLE foobar( name TEXT NOT NULL, id SERIAL)')
+  client.query('BEGIN')
```

Comment:
s/client/await client/

Reply:
ah yeah that's a hold over when i thought it might have been related to the query queue (which is a horrible unfortunate API design mistake I made 10 years ago).  So I initially wrote the test kinda abusing that feature thinking it was something I'd done.  I just added await back here, thanks!